platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -518,6 +518,8 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		17C4564F1E4E19BB0029C9B1 /* darwin_conversion.hpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.h; path = darwin_conversion.hpp; sourceTree = "<group>"; };
```

Comment:
Remember to add this file to macos.xcodeproj as well, since all the runtime styling code is shared between both platforms.